src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDataService.java

```diff
@@ -128,7 +128,7 @@ public CaseData getCase(UUID caseUUID) {
         return caseData;
     }
 
-    private CaseData getCaseData(UUID caseUUID) {
+    protected CaseData getCaseData(UUID caseUUID) {
```

Comment:
We were auditing a case view here and we don't need to if we use the 'internal' method